src/ocean/math/WideUInt.d

```diff
@@ -336,7 +336,7 @@ public struct WideUInt ( size_t N )
 
     ***************************************************************************/
 
-    public void opAddAssign ( uint rhs )
+    public void opOpAssign ( string op ) ( uint rhs ) if (op == "+")
```

Comment:
This is covered here: https://github.com/sociomantic-tsunami/ocean/blob/v5.x.x/src/ocean/math/WideUInt.d#L349

Reply:
ditto